src/main/java/org/prebid/server/bidder/between/BetweenBidder.java

```diff
@@ -0,0 +1,151 @@
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * Between {@link Bidder} implementation.
+ */
+public class BetweenBidder implements Bidder<BidRequest> {
+
+    private static final TypeReference<ExtPrebid<?, ExtImpBetween>> BETWEEN_EXT_TYPE_REFERENCE =
+            new TypeReference<ExtPrebid<?, ExtImpBetween>>() {
+            };
+    private static final String URL_HOST_MACRO = "{{Host}}";
+
+    private final String endpointUrl;
+    private final JacksonMapper mapper;
+
+    public BetweenBidder(String endpointUrl, JacksonMapper mapper) {
+        this.endpointUrl = HttpUtil.validateUrl(Objects.requireNonNull(endpointUrl));
+        this.mapper = Objects.requireNonNull(mapper);
+    }
+
+    @Override
+    public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest request) {
+
+        final List<BidderError> errors = new ArrayList<>();
+        final Map<Imp, ExtImpBetween> validImpsWithExts = new HashMap<>();
+
+        for (Imp imp : request.getImp()) {
+            try {
+                final ExtImpBetween extImpBetween = parseImpExt(imp);
+                validImpsWithExts.put(imp, extImpBetween);
+            } catch (PreBidException e) {
+                errors.add(BidderError.badInput(e.getMessage()));
+            }
+        }
+
+        if (validImpsWithExts.size() == 0) {
+            return Result.emptyWithError(BidderError.badInput("No valid Imps in Bid Request"));
+        }
+
+        final List<HttpRequest<BidRequest>> madeRequests = new ArrayList<>();
+
+        for (Map.Entry<Imp, ExtImpBetween> entry : validImpsWithExts.entrySet()) {
+            madeRequests.add(makeSingleRequest(entry.getValue(), request, entry.getKey()));
+        }
+
+        return Result.of(madeRequests, errors);
+    }
+
+    private HttpRequest<BidRequest> makeSingleRequest(ExtImpBetween extImpBetween, BidRequest request, Imp imp) {
+
+        final String url = endpointUrl.replace(URL_HOST_MACRO, extImpBetween.getHost());
+        final BidRequest outgoingRequest = request.toBuilder().imp(Collections.singletonList(imp)).build();
+
+        return
+                HttpRequest.<BidRequest>builder()
+                        .method(HttpMethod.POST)
+                        .uri(url)
+                        .headers(HttpUtil.headers())
+                        .payload(outgoingRequest)
+                        .body(mapper.encode(outgoingRequest))
+                        .build();
+    }
+
+    private ExtImpBetween parseImpExt(Imp imp) {
+        final ExtImpBetween extImpBetween;
+        try {
+            extImpBetween = mapper.mapper().convertValue(imp.getExt(), BETWEEN_EXT_TYPE_REFERENCE).getBidder();
+        } catch (IllegalArgumentException e) {
+            throw new PreBidException(String.format("Missing bidder ext: %s", e.getMessage()));
+        }
+        if (StringUtils.isEmpty(extImpBetween.getHost())) {
```

Comment:
Personally i don't like `StringUtils.isBlank(..)` but seems here it is more desirable.